README.md

```diff
@@ -39,6 +39,8 @@ Then:
 Head to the address it gives you to check if your changes have worked. More information about using mkdocs can be found [here](http://www.mkdocs.org/)
 
 ## Release Notes
+- **4.2.1, Jan 2023**
+  * Bug fixes
```

Comment:
Is there nothing other than bug fixes added since last release?

Reply:
@mattdeshAQ can you make this 4.3.0

Reply:
My bad, I didn't realise there were so many other commits which will be part of the new version.

Reply:
This has been updated now